builtin/credential/aws/path_login.go

```diff
@@ -1353,7 +1353,7 @@ func parseIamArn(iamArn string) (*iamEntity, error) {
 		return nil, fmt.Errorf("unrecognized arn: contains %d colon-separated parts, expected 6", len(fullParts))
 	}
 	if fullParts[0] != "arn" {
-		return nil, fmt.Errorf("unrecognized arn: does not begin with arn:")
+		return nil, fmt.Errorf("unrecognized arn: does not begin with arn")
```

Comment:
Actually, the : is correct here -- the check is to see if the string begins with "arn:". Perhaps this might be clearer?

```go
return nil, fmt.Errorf("unrecognized arn: does not begin with \"arn:\"")
```

Reply:
Thanks Joel! Good point, I'll update it just like that.